pom.xml

```diff
@@ -150,7 +150,7 @@ Copyright (c) 2010, 2015, Oracle and/or its affiliates. All rights reserved.
             <plugin>
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
-                <version>0.7.4</version>
```

Comment:
d'oh!
I forgot to move this up, we actually do support and run on jacoco 0.7.5 and looking at jacoco pages, we can just go to 0.7.7 (I will retest in our internal farm and let you know)